packages/yourloops/pages/training/training.tsx

```diff
@@ -124,22 +134,55 @@ function TrainingPage(): JSX.Element {
               </Box>
             </CardMedia>
             <CardContent className={classes.cardContent}>
-              <Trans
-                i18nKey="training-body"
-                t={t}
-                components={{ trainingLink }}
-                values={{ training }}>
-                New {{ training }} available
-              </Trans>
+              {trainingOpened
+                ? <FormControlLabel
+                  id={'form-control-training'}
+                  control={
+                    <Checkbox
+                      id={'checkbox-training'}
+                      aria-label={t('training-checkbox')}
+                      className={classes.checkbox}
+                      checked={checked}
+                      onChange={() => {
+                        setChecked(true)
+                      }}
+                      name="training"
+                      color="primary"
+                    />
+                  }
+                  label={t('acknowledge-training')}
+                  className={classes.formControlLabel}
+                />
+                : t('training-body', { training })
+              }
               <div className={classes.buttons}>
-                <Button
-                  variant="contained"
-                  color="primary"
-                  disableElevation
-                  onClick={ackTraining}
-                >
-                  {t('button-acknowledge')}
-                </Button>
+                {trainingOpened
+                  ? <Button
+                    variant="contained"
+                    color="primary"
+                    disableElevation
+                    onClick={ackTraining}
+                    disabled={!checked}
+                  >
+                    {t('button-accept')}
+                  </Button>
+                  : <Link
+                    underline="none"
+                    aria-label={training}
+                    href={diabeloopExternalUrls.training(user?.role)}
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    <Button
+                      variant="contained"
+                      color="primary"
+                      disableElevation
+                      onClick={openTraining}
+                    >
+                      {t('open-training', { training })}
```

Comment:
Could be removed now it's no more in translation
```suggestion
                      {t('open-training')}
```